src/show/pages/public/CreateProject/Subviews/index.js

```diff
@@ -0,0 +1,443 @@
+import ReactDatePicker from 'react-datepicker';
```

Comment:
- comment for the whole file - 

Every step should live in his own js file, the imported into the `subviews/index.js` file then exported from there (just like we do with the components). This brings better readability and maintainability.

Reply:
resolve at standup